### ISSUE TITLE
rMPP?

### ISSUE THREAD

**Issue by folofjc** [Evidlo/remarkable_entware#29]
Has only tried to install this on the rM Paper Pro?

First, it appears that even in developer mode, the file system is read only? I had to remount it as `rw` to even be able to download the install script.

Then when I try to install, I get this:

```
# wget -O - http://raw.githubusercontent.com/Evidlo/remarkable_entware/master/install.sh | sh                           
Connecting to raw.githubusercontent.com (185.199.109.133:80)                                                                                 
Connecting to raw.githubusercontent.com (185.199.108.133:443)                                                                               
 wget: note: TLS certificate validation not implemented                                                                                       
writing to stdout                                                                                                                            -                    100% |*********************************************************************************************|  3736  0:00:00 ETA 
written to stdout                                                                                                                            
Info: Checking for prerequisites and creating folders...                                                                                     
Created symlink /etc/systemd/system/local-fs.target.wants/opt.mount → /etc/systemd/system/opt.mount.                                         
Info: Opkg package manager deployment...                                                                                                     
Connecting to bin.entware.net (158.160.45.54:80)                                                                                             
saving to '/opt/bin/opkg'                                                                                                                    
opkg                 100% |*********************************************************************************************|  734k  0:00:00 ETA 
'/opt/bin/opkg' saved                                                                                                                        
Connecting to bin.entware.net (158.160.45.54:80)                                                                                             
saving to '/opt/etc/opkg.conf'                                                                                                               
opkg.conf            100% |*********************************************************************************************|   171  0:00:00 ETA 
'/opt/etc/opkg.conf' saved                                                                                                                   
Connecting to bin.entware.net (158.160.45.54:80)                                                                                             
saving to '/opt/lib/ld-2.27.so'                                                                                                              
ld-2.27.so           100% |*********************************************************************************************|  131k  0:00:00 ETA 
'/opt/lib/ld-2.27.so' saved                                                                                                                  
Connecting to bin.entware.net (158.160.45.54:80)                                                                                             
saving to '/opt/lib/libc-2.27.so'                                                                                                            
libc-2.27.so         100% |*********************************************************************************************| 1218k  0:00:00 ETA 
'/opt/lib/libc-2.27.so' saved                                                                                                                
Connecting to bin.entware.net (158.160.45.54:80)                                                                                             
saving to '/opt/lib/libgcc_s.so.1' 
libgcc_s.so.1        100% |*********************************************************************************************| 46752  0:00:00 ETA 
'/opt/lib/libgcc_s.so.1' saved                                                                                                               
Connecting to bin.entware.net (158.160.45.54:80)                                                                                             
saving to '/opt/lib/libpthread-2.27.so'                                                                                                      
libpthread-2.27.so   100% |*********************************************************************************************| 92656  0:00:00 ETA '/opt/lib/libpthread-2.27.so' saved                                                                                                          
cat: read error: Is a directory                                                                                                              
Computed hash did not match.
```

I assume that the hash error is similar to #26, but what is the `Is a directory`? The `libpthread-2.27.so` is indeed a file. What is the `cat` trying to read?

**Comment by hmenzagh** [comment]
Just made this: https://github.com/hmenzagh/rmpp-entware

**Comment by Evidlo** [comment]
Fixed with #27 hopefully
